Source/JavaScriptCore/inspector/agents/InspectorScriptProfilerAgent.cpp

```diff
@@ -122,22 +128,30 @@ Seconds InspectorScriptProfilerAgent::willEvaluateScript()
 {
     m_activeEvaluateScript = true;
 
+    CheckedPtr environment = m_environment.get();
+    if (!environment)
```

Comment:
This should be makeUnexpected to match the other case where the environment is gone, i think?